### PR TITLE
MOR-267: Updated Dashboard content and design according to changes proposed in MOR-145

### PR DESCRIPTION
**Pull request by javihernandez**
See https://morphic.atlassian.net/browse/MOR-145

### CODE REVIEW

**Comment by kaspermarkus** [comment]
Conflicts after latest merges.. can you update @javihernandez

**Comment by javihernandez** [comment]
> Conflicts after latest merges.. can you update @javihernandez

updated now

File: src/views/Dashboard.vue [hunk 1]
```diff
@@ -182,37 +141,25 @@ export default {
...
   },
-  mounted () {
+  mounted: function () {
     this.loadData()
+    // This is required to re-draw the arrows, sorry :P
+    let that = this
+    this.$nextTick(function() {
+      window.addEventListener("resize", function () {that.$forceUpdate()})
+    }(that))
+  },
```
Comment (kaspermarkus) [inline src/views/Dashboard.vue:151]: I think with fancy modern javascript, you can use the arrow syntax which will preserve the "this"... like so:

```
this.$nextTick(() => window.addEventListener("resize", () => this.$forceUpdate)))
```

**Review by kaspermarkus — CHANGES_REQUESTED** [review]
Minor change, just for cleanness, but otherwise works perfectly for me

**Review by kaspermarkus — APPROVED** [review]

File: src/views/Dashboard.vue (hunk 1 above)
Comment (javihernandez) [reply src/views/Dashboard.vue:151]: Actually I tested this (and other alternatives) and couldn't make it work :man_shrugging:

**Pull request merged**